appveyor.yml

```diff
@@ -27,3 +27,4 @@ install:
 test_script:
   - php .\php\composer.phar install
   - vendor/bin/phpunit.bat --coverage-text
+  - vendor/bin/phpstan.bat analyse
```

Comment:
It is not really required to have phpstan running on multiple platforms. Phpstan won't detect any differences based on the os.

Reply:
Removed. Ironically, I only added this because I thought someone would have a comment if I didn't haha.